pkg/tests/apis/helper.go

```diff
@@ -327,20 +324,27 @@ func (c *K8sTestHelper) LoadYAMLOrJSON(body string) *unstructured.Unstructured {
 	return &unstructured.Unstructured{Object: unstructuredMap}
 }
 
-func (c K8sTestHelper) createTestUsers(orgId int64) OrgUsers {
+func (c K8sTestHelper) createTestUsers(orgName string) OrgUsers {
 	c.t.Helper()
 
 	store := c.env.SQLStore
-	store.Cfg.AutoAssignOrg = true
-	store.Cfg.AutoAssignOrgId = int(orgId)
+	defer func() {
+		store.Cfg.AutoAssignOrg = false
+		store.Cfg.AutoAssignOrgId = 1 // the default
+	}()
+
 	quotaService := quotaimpl.ProvideService(store, store.Cfg)
 
 	orgService, err := orgimpl.ProvideService(store, store.Cfg, quotaService)
 	require.NoError(c.t, err)
 
-	gotID, err := orgService.GetOrCreate(context.Background(), fmt.Sprintf("Org%d", orgId))
-	require.NoError(c.t, err)
-	require.Equal(c.t, orgId, gotID)
+	orgId := int64(1)
+	if orgName != "Org1" {
+		orgId, err = orgService.GetOrCreate(context.Background(), orgName)
```

Comment:
the details on how org1 vs other orgs are create in tests is pretty gnarly and has many special cases across various services.  The Org1 vs OrgB is odd -- but does test our "default" vs "org-XYZ" case 🤷🏻